### ISSUE TITLE
Modal shifting background to the right

### ISSUE THREAD

**Issue by kobeman** [rafaelpimpa/buefy#358]
**Buefy** version: v0.5.3
**Vuejs** version: v2.4.4
**OS/Browser**: Mac/Chrome

### Description

When opening a modal, my page content shifts to the right by about 15px (actually removing scrollbar space).

### Steps to reproduce

Example at buefy.github.io/#/documentation/modal

### Expected behavior

I would like my background to stay static. 

### Actual behavior

OnOpen background shifts to the right,

All help appreciated thank you,
Roger

**Comment by rafaelpimpa** [comment]
The page gets clipped (``overflow: hidden``) when the modal appears, to prevent weird behavior of the background. I'm not sure if there's other way to do that.

**Comment by kobeman** [comment]
Hi Rafael,

I have tried numerous options including adding a right-margin at the 1024px break:

`@media screen and (min-width: 1024px) {
  html.is-clipped {
    margin-right: 15px !important;
  }
}`

While this resolves my Mac Safari/Chrome (1920x1080) issue it then creates a 15px shift in my chromebook (1366x768) which wasn't there before.

Interesting is that Vue-Blu for example does not produce the scrollbar shift in any browser/resolutions? I have looked at their code but am unabled to port a fix.

The shift is really a question of preference and not a bug. My DigitalOcean user ui has a shift while Dropbox doesn't... I will keep trying and post an update if I find a global solution for those interested.

Thank you for all the great work on Buefy!
Roger
